components/CreateOrganization.js

```diff
@@ -149,4 +166,4 @@ const addCreateOrganizationMutation = graphql(createOrganizationMutation, {
 
 const addGraphql = compose(addCreateOrganizationMutation, addEditCollectiveMembersMutation);
 
-export default addGraphql(CreateOrganization);
```

Comment:
Is this `injectIntl` + `intl` propType necessary? I don't see where it's being used